project-reports/2024/2024-Q4-Hyperledger-Identus.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Identus
+parent: 2024
+grand_parent: Project Updates
+nav_exclude: true
+---
+
+Created by Javier Ribó with input from other maintainers
+
+
+# Project Health
+
+The project is still new to the Hyperledger Ecosystem and continues to have steady growth of contributors.
+The project can now be found on the LFX dashboard which can now be checked for health insights. In the last quarter we have seen an increase on the number of github issues created while the active contributors have gone down 8%.
+
+
+# Questions/Issues for the TOC
+
+Is there a way to access funds for open source contributions and development? 
```

Comment:
where did this question come from?

Reply:
From the previous report in Identus, removed it anyway its now showing N/A